EcoSoapBank/Info.plist

```diff
@@ -83,8 +83,6 @@
 	</dict>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
-	<key>UIMainStoryboardFile</key>
-	<string>Main</string>
```

Comment:
I'm actually kind of surprised things were working without this taken out. Thanks for doing that.